src/github/pullRequestManager.ts

```diff
@@ -109,16 +110,19 @@ export class PullRequestManager {
 	constructor(
 		private _repository: Repository,
 		private readonly _telemetry: ITelemetry,
+		private _credentialStore: CredentialStore = new CredentialStore(_telemetry),

```

Comment:
There are some tests where I was able to allow a `PullRequestManager` to create its own `CredentialStore` and access it later, and others where I had to create it ahead of time and pass it in. It's likely possible to do some later refactoring to unify them.

By the way, the complication here is that `CredentialStore` registers a command in its constructor, so you can't instantiate more than one instance at a time.